src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -0,0 +1,87 @@
+package cromwell.binding
+
+import com.google.api.services.genomics.model.Disk
+import cromwell.binding.AstTools.{AstNodeName, EnhancedAstNode}
+import cromwell.binding.RuntimeAttributes.{Defaults, LocalDisk}
+import cromwell.parser.MemorySize
+import cromwell.parser.WdlParser.{Ast, AstList}
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+object RuntimeAttributes {
+  /** Convenience class to convert to Google `Disk` class. */
+  private case class LocalDisk(name: String, sizeGb: Long, diskType: String) {
+    def toDisk: Disk = new Disk().setSizeGb(sizeGb).setType(diskType).setName(name)
+  }
+
+  /** Fallback values if values for these keys are not specified in a task "runtime" stanza. */
+  object Defaults {
+    val Cpu = 1
+    val Disk = Seq(LocalDisk("local-disk", 100L, "LOCAL_SSD").toDisk)
+    val FailOnStderr = false
+    val MemoryInBytes = MemorySize.GB.toBytes(2)
+    val Preemptible = false
+    val Zones = Array("us-central1-a")
+  }
+}
+
+/**
+ * Parses configuration values per WDL/JES specs.
+ */
+case class RuntimeAttributes(ast: Ast) {
```

Comment:
Since the `ast` isn't being used anywhere post construction perhaps have an `apply` which takes it in and chucks it when it makes the real `RuntimeAttributes`. 

Combining my two comments (despite them not happening in the correct order top to bottom) if `foo()` was in the companion object you could then call something like `RuntimeAttributes(foo(ast))` with `case class RuntimeAttributes(attributes: Map[String, String])`